### PR TITLE
Add preferred pronouns

### PR DESCRIPTION
**Pull request by angelo-v**

### CODE REVIEW

**Comment by angelo-v** [comment]
See https://gitter.im/solid/solidos?at=60ad54b278e1d6477d65145a

please add unit tests for the changes in presenter and add an integration test for the display of the pronouns

File: src/presenter.ts [hunk 1]
```diff
@@ -17,29 +19,43 @@ export const presentProfile = (
...
       : null;
   const locality =
     address != null
-      ? store.anyValue(address as NamedNode, ns.vcard("locality"))
+      ? store.anyValue(address as NamedNode, ns.vcard("locality"), null, profile)
       : null;
 
   const { backgroundColor, highlightColor } = getColors(subject, store);
 
+  let pronouns = store.anyJS(subject, ns.solid('preferredSubjectPronoun'), null, profile) || ''
```
Comment (angelo-v) [inline src/presenter.ts:40]: This is all about pronouns and should be extracted to its own function with a respective name. This function could then be unit tested on its own as well

File: src/presenter.ts [hunk 2]
```diff
@@ -17,10 +19,11 @@ export const presentProfile = (
   subject: NamedNode,
   store: IndexedFormula
 ): ProfilePresentation => {
+  const profile = subject.doc()
   const name = utils.label(subject);
   const imageSrc = findImage(subject);
-  const role = store.anyValue(subject, ns.vcard("role"));
-  const orgName = store.anyValue(subject, ns.vcard("organization-name"));
+  const role = store.anyValue(subject, ns.vcard("role"))
```
Comment (SharonStrats) [inline src/presenter.ts:25]: just missing a semicolon

File: src/presenter.ts [hunk 3]
```diff
@@ -34,12 +37,25 @@ export const presentProfile = (
 
   const { backgroundColor, highlightColor } = getColors(subject, store);
 
+  let pronouns = store.anyJS(subject, ns.solid('preferredSubjectPronoun')) || ''
```
Comment (SharonStrats) [inline src/presenter.ts:40]: I think angelo mentioned making this a function.  The weird thing is I thought I saw the function, but I can't seem to find it.

**Review by SharonStrats — CHANGES_REQUESTED** [review]
It looks great!!  I tested it by changing the webid to yours.

**Review by SharonStrats — APPROVED** [review]
I've added a test for integration to check pronouns

**Review by SharonStrats — CHANGES_REQUESTED** [review]
Actually I realized we need to do 2 more tests.  I will have to add them later.

**Review by SharonStrats — APPROVED** [review]
Okay, got it.  integration and unit test.

**Pull request merged**